### PR TITLE
Elad lbm1 35358/add readme for pr checks

### PR DESCRIPTION
**Pull request by elada-lb**
This PR adds a readme for how to run a PR check in the discovery-client repository.

Issue: LBM1-35358

## Summary by CodeRabbit

- **New Features**
	- Added a "PR checks" section in the `README.md` linking to a new guide for pull request workflows.
	- Introduced a detailed guide for running pull request checks for both organization and non-organization users in `docs/pr-check.md`.

### CODE REVIEW

**Review by evyatar-lb — APPROVED** [review]
Nice!

**Review by shaharz-lb — APPROVED** [review]
It looks good! 
Two comments:
1. In the [private doc](https://github.com/LightBitsLabs/eng-docs/blob/eladLBM1-35358/add-readme-for-pr-checks/pr-checks/public-repository-pr-checks.md): Please include a warning before initiating the PR check to confirm that the code is secure and free of any security vulnerabilities (This comment should be inside the private doc only).
2. Consider adding the content of this MD to the main README.md to ensure everyone knows how to open a PR.

File: README.md [hunk 1]
```diff
@@ -307,6 +308,21 @@ yum -q makecache -y --disablerepo='*' --enablerepo='lightbits-discovery-client'
...
+
+### How-to
+
+1. **Non-Organization Users:**
+   - Fork the repository
+   - Make the change you want
+   - Open a pull request with the public repo (not the fork) as the base
+   - A person inside the organization will review the pull request and, if it looks fine, he'll trigger the PR check on the pull request branch
+   - You'll be notified with the results of the PR check once it's done
+2. **Organization Users:**
```
Comment (muliby-lb) [inline README.md:321]: Lightbits Contributors

File: README.md [hunk 2]
```diff
@@ -307,6 +308,21 @@ yum -q makecache -y --disablerepo='*' --enablerepo='lightbits-discovery-client'
...
 
+## PR Checks
+
+### How-to
+
+1. **Non-Organization Users:**
+   - Fork the repository
+   - Make the change you want
+   - Open a pull request with the public repo (not the fork) as the base
+   - A person inside the organization will review the pull request and, if it looks fine, he'll trigger the PR check on the pull request branch
```
Comment (muliby-lb) [inline README.md:319]: Change "a person inside the org" to "a committer from Lightbits"
Change "he" to "they"

**Pull request merged**